README.md

```diff
@@ -24,6 +24,8 @@ Flyte Admin has a few useful make targets for linting and testing. Please use th
 minor bugs and linting errors.
 
 ```
+  # Please make sure you have goimports installed, i.e.
+  $ go install golang.org/x/tools/cmd/goimports@latest
```

Comment:
you can actually run `make install` which should download all the go tooling you need to run goimports